tsl/src/chunk_api.c

```diff
@@ -1627,3 +1629,81 @@ chunk_api_get_chunk_relstats(PG_FUNCTION_ARGS)
 {
 	return chunk_api_get_chunk_stats(fcinfo, false);
 }
+
+Datum
+chunk_api_copy_data(PG_FUNCTION_ARGS)
```

Comment:
Nit: Usually, `chunk_api` contains functions that are executed on the DNs, not the functions called from the AN.